src/plot/ROOTMultiPlot.cpp

```diff
@@ -77,7 +77,10 @@ ROOTMultiPlot::Construct(){
   for(size_t i = 0; i < fNames.size(); i++){
     fLegend.AddEntry(&fHists.at(i), fNames.at(i).c_str());
     fHists[i].SetLineColor(fColorScheme[i %fNcolors]);
-    fHists.at(i).Draw("same");
+    if(i!=0)
```

Comment:
Wait, I remember the reason for this now (although maybe there isn't a use case): what if construct is called multiple times?  At least a clear is needed on the canvas here, right?